workflow-runtime/src/jsTest/kotlin/com/squareup/workflow1/ErrorLoggingInterceptor.kt

```diff
@@ -0,0 +1,22 @@
+package com.squareup.workflow1
+
```

Comment:
I can't see what's different here that we are requiring to expect/actual these? Is it in a later commit?

Reply:
it's subtle but js uses `simpleName` instead of `qualifiedName` in the other targets